### DISCUSSION TITLE
Update from 22.04 to 22.10

### DISCUSSION THREAD

**Discussion by jbeltran73-2** [linux-surface/linux-surface#968] [Q&A] [answered]
Is it counterproductive to update to version 22.10 of Ubuntu? If it is done, does it interfere with the update of the Kernel from this Github? Thank you very much!

**Comment by qzed** [comment, marked as answer]
Updating should work. It might be possible that the linux-surface repo gets deactivated during the update, so you might want to check that it's still there after updating (and if necessary re-activate/re-add it), but otherwise I don't think you have to do anything special.

**Reply by jbeltran73-2** [reply]
Wow, that was a fast answer. Then I'll try it. Will come back with the results. Thank you!

**Reply by jbeltran73-2** [reply]
Done. No problems, at all. 
![image](https://user-images.githubusercontent.com/55049032/197402164-8407c9d6-38a1-417d-8ede-7add2760727f.png)

**Reply by NP-chaonay** [reply]
> Done. No problems, at all. ![image](https://user-images.githubusercontent.com/55049032/197402164-8407c9d6-38a1-417d-8ede-7add2760727f.png)

on Ubuntu, if nothing is set, the default is all repo except ubuntu distro repo (if I not wrong 100%) is disabled
and no automatically enabled.

so afterward of upgrade, recommmended not to let upgrader auto remove unused kernel (must do; if you have disk encryptioned on unless you have external USB keyboard, because if you boot into new kernel/configuration, the surface kernel may not able to use to unlock disk encryption due to driver config)

and reboot it and reenable the linux-surface repo and then apt update +apt upgrade

**Reply by jbeltran73-2** [reply]
I did tell the system to remove all unnecessary and it's working like a charm. No problems at all. I did reboot and I didn't touch or modify anything.

**Reply by NP-chaonay** [reply]
> I did tell the system to remove all unnecessary and it's working like a charm. No problems at all. I did reboot and I didn't touch or modify anything.

yeah it depends on cases I just count on worst case, but average case (without disk encryptioned at least) should go not fatal.
